### PR TITLE
Login signup

### PR DESCRIPTION
**Pull request by Omkar-Ghate**
-- For login and signup components. Not yet responsive. Please review the code and design
-- On the login page, click on Create new account link to go to the signup page 
-- On sign up page, click on Already have an account link to go to the login page

### CODE REVIEW

File: pages/login/login.css [hunk 1]
```diff
@@ -0,0 +1,338 @@
...
+  justify-content: center;
+  align-items: center;
+}
+.nav-icon-link .nav-icon-text {
+  font-size: 0.75rem;
+  margin-top: 4px;
+  font-family: "Poppins", sans-serif;
+}
+.nav-icon-link .nav-icon {
+  color: #696b79;
```
Comment (dev-prabhat) [inline pages/login/login.css]: use var to define color value

File: pages/login/login.css [hunk 2]
```diff
@@ -0,0 +1,338 @@
...
+.nav-bar-links {
+  display: flex;
+  z-index: 5;
+}
+
+.nav-icons {
+  display: flex;
+  justify-self: right;
+}
+.nav-icon-link {
```
Comment (dev-prabhat) [inline pages/login/login.css:62]: spacing is irregular

**Review by dev-prabhat — CHANGES_REQUESTED** [review]
It looks good to me but some changes are required , I have also mentioned

File: pages/login/login.css [hunk 3]
```diff
@@ -0,0 +1,338 @@
...
+
+.burger {
+  display: none;
+  z-index: 6;
+  cursor: pointer;
+}
+.burger .line1, .burger .line2, .burger .line3 {
+  height: 2px;
+  width: 1.5rem;
+  background-color: #424553;
```
Comment (rutu14) [inline pages/login/login.css]: Color variable can be applied.

**Review by rutu14 — APPROVED** [review]

**Review by dev-prabhat — APPROVED** [review]
Good to go

**Pull request merged**